github_connector_odoo/models/odoo_module_version.py

```diff
@@ -286,13 +289,30 @@ def _compute_organization_serie_id(self):
             module_version.organization_serie_id =\
                 res and res[0].id or False
 
+    @api.model
+    def get_module_category(self, info):
+        """Used to search the category of the module by the data given
+        on the manifest.
+
+        :param dict info: The parsed dictionary with the manifest data.
+        :returns: recordset of the given category if exists.
+        """
+        category_obj = self.env['odoo.category']
+        category = info.get('category', False)
+        other_categ = category_obj.search([('name', '=', 'Other')], limit=1)
+        if not category:
+            return other_categ
+        found_category = category_obj.search([('name', '=', category)], limit=1)
+        return found_category or other_categ
+
     # Custom Section
     @api.model
     def manifest_2_odoo(self, info, repository_branch, module):
         author_list =\
             (type(info['author']) == list) and info['author'] or\
             (type(info['author']) == tuple) and [x for x in info['author']] or\
             info['author'].split(',')
+        category = self.get_module_category(info)
         res = {
```

Comment:
You can rewrite this to:
`return { ... }`

Also since the category variable is only used here, I think it's better to only return the ID of the record from the function
Then you would get:
`return {
...
'category_id': self.get_module_category(info)
...
}`